pydm/widgets/related_display_button.py

```diff
@@ -43,6 +43,11 @@ def __init__(self, parent=None, filename=None):
             PyDMRelatedDisplayButton.RULE_PROPERTIES.update(
                 {'Text': ['setText', str]})
 
+        if 'Filename' not in PyDMRelatedDisplayButton.RULE_PROPERTIES:
+            PyDMRelatedDisplayButton.RULE_PROPERTIES = PyDMPrimitiveWidget.RULE_PROPERTIES.copy()
```

Comment:
Setting this to a copy of the PyDMPrimitiveWidget rules will cause the text rule added right above this to get lost. Will want to make sure both rules get added, maybe by just copying PyDMRelatedDisplayButton.RULE_PROPERTIES, or however you'd like.